src/transformers/models/wav2vec2/convert_wav2vec2_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -20,34 +20,39 @@
 import fairseq
 import torch
 
-from transformers import Wav2Vec2Config, Wav2Vec2ForCTC, logging
+from transformers import Wav2Vec2Config, Wav2Vec2ForCTC, Wav2Vec2Model, logging
 
 
 logging.set_verbosity_info()
 logger = logging.get_logger(__name__)
 
 MAPPING = {
-    "post_extract_proj": "wav2vec2.feature_projection.projection",
-    "encoder.pos_conv.0": "wav2vec2.encoder.pos_conv_embed.conv",
-    "self_attn.k_proj": "wav2vec2.encoder.layers.*.attention.k_proj",
-    "self_attn.v_proj": "wav2vec2.encoder.layers.*.attention.v_proj",
-    "self_attn.q_proj": "wav2vec2.encoder.layers.*.attention.q_proj",
-    "self_attn.out_proj": "wav2vec2.encoder.layers.*.attention.out_proj",
-    "self_attn_layer_norm": "wav2vec2.encoder.layers.*.layer_norm",
-    "fc1": "wav2vec2.encoder.layers.*.feed_forward.intermediate_dense",
-    "fc2": "wav2vec2.encoder.layers.*.feed_forward.output_dense",
-    "final_layer_norm": "wav2vec2.encoder.layers.*.final_layer_norm",
-    "encoder.layer_norm": "wav2vec2.encoder.layer_norm",
-    "w2v_model.layer_norm": "wav2vec2.feature_projection.layer_norm",
+    "post_extract_proj": "feature_projection.projection",
+    "encoder.pos_conv.0": "encoder.pos_conv_embed.conv",
+    "self_attn.k_proj": "encoder.layers.*.attention.k_proj",
```

Comment:
Will the weights on the hub need to be updated?

Reply:
No it was only for new models